src/Behat/Testwork/Output/Printer/JUnitOutputPrinter.php

```diff
@@ -151,11 +152,17 @@ public function setFileName($fileName, $extension = 'xml')
     public function flush()
     {
         if($this->domDocument instanceof \DOMDocument){
-            $this->getWritingStream()->write(
-                $this->domDocument->saveXML(null, LIBXML_NOEMPTYTAG),
-                false,
-                OutputInterface::OUTPUT_RAW
-            );
+            try {
```

Comment:
And in the junit formatter we catch this exception and throw a more specific error